grafana/scylla-advanced.template.json

```diff
@@ -27,6 +27,81 @@
                 ],
                 "title": "New row"
             },
+            {
+                "class": "row",
+                "height": "25px",
+                "panels": [
+                    {
+                        "class": "percentunit_panel",
+                        "span": 3,
+                        "targets": [
+                            {
+                                "expr": "sum(rate(scylla_io_queue_consumption{class=~\"$classes\", instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\"}[1m])) by ([[by]], class)",
+                                "intervalFactor": 1,
+                                "legendFormat": "{{dc}} {{node}} {{class}}",
+                                "metric": "seastar_io_queue_delay",
+                                "refId": "A",
+                                "step": 30
+                            }
+                        ],
+                        "fieldConfig": {
+					        "defaults": {
+					          "class": "fieldConfig_defaults",
+					          "custom": {
+					             "class":"fieldConfig_defaults_custom",
+					             "axisSoftMax": 1,
+					             "stacking": {
+						          "mode": "normal",
+						          "group": "A"
+						        }
+					          },
+					          "unit": "percentunit"
+					        },
+					        "overrides": []
+					     },
+                        "options": {
+                            "class":"desc_tooltip_options"
+                        },
+                        "description": "seastar_io_queue_delay",
+                        "title": "I/O Queue consumption by [[by]]"
+                    },
+                   {
+                        "class": "percentunit_panel",
+                        "span": 3,
+                        "targets": [
+                            {
+                                "expr": "sum(rate(scylla_io_queue_adjusted_consumption{class=~\"$classes\", instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\"}[1m])) by ([[by]], class)",
```

Comment:
This one -- the adjusted consumption can be omitted to save space on the dashboard